src/style.css

```
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  padding: 0;
+  margin: 20px 8px 0;
+}
+
+.form--big {
+  height: 70px;
+}
+
+.form--small {
+  height: 42px;
+}
```

```suggestion
.form {
  padding: 0;
  margin: 20px 8px 0;
  height: 70px;
}

.form--small {
  height: 42px;
}
```
and here we just override differences
you can do in the same way for another selectors